pegasus/helpers/form_helpers.rb

```diff
@@ -195,6 +199,7 @@ def update_form(kind, secret, data)
   form[:reviewed_at] = nil
   form[:reviewed_by] = nil
   form[:reviewed_ip] = nil
+  form[:hashed_email] = Digest::MD5.hexdigest(form[:email]) if data.key?(:email_s)
```

Comment:
My vote would be to define `normalized_email` and use it everywhere (for `forms[:email]` and `forms[:hashed_email]`).